Warning, the following versions were found for oauthlib
0.7.2
0.7.1
0.7.0
0.6.3
0.6.2
0.6.1
0.6.0
0.5.1
0.5.0
0.4.2
0.4.1
0.4.0
0.3.8
0.3.7
0.3.6
0.3.5
0.3.4
0.3.3
0.3.2
0.3.0
0.1.3
0.1.2
0.1.1
0.1.0
0.0.2
0.0.1
Using 0.7.2
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/o/oauthlib/oauthlib-0.7.2.tar.gz (104 KB) for oauthlib.
Downloading oauthlib (use --no-download to skip this step)
Using cached download
Unpacking oauthlib...
done
working in /tmp/tmpswqTkXconda_skeleton_oauthlib-0.7.2.tar.gz
Fetching package metadata: ......
Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).
Traceback (most recent call last):
  File "setup.py", line 12, in <module>
    from setuptools import setup, find_packages
ImportError: No module named setuptools

The following NEW packages will be INSTALLED:

    numpy:  1.9.2-py27_0
    pyyaml: 3.11-py27_0 

Linking packages ...
[                    ]|                                                  |   0%[numpy               ]|                                                  |   0%[pyyaml              ]|#########################                         |  50%[      COMPLETE      ]|##################################################| 100%
Applying patch: '/tmp/tmpswqTkXconda_skeleton_oauthlib-0.7.2.tar.gz/pypi-distutils.patch'
$PYTHONPATH = /tmp/tmpswqTkXconda_skeleton_oauthlib-0.7.2.tar.gz/oauthlib-0.7.2
Error: command failed: /home/ubuntu/miniconda/envs/_build/bin/python setup.py install
